### DISCUSSION TITLE
Sentinel data update frequency

### DISCUSSION THREAD

**Discussion by edornd** [microsoft/PlanetaryComputer#368] [Q&A]
Hi! I'd like to use the Planetary STAC API to build time series of Sentinel data (S1 and S2 mostly) periodically.
I was wondering what's the update frequency of the catalog, i.e., how much delay is there from the satellite acquisition to the catalog insertion? For instance, if I know that the catalog is updated once a week, I'll make sure to schedule the task with a delay of 7 days just to avoid missing some info.
Thanks a lot!

**Comment by TomAugspurger** [comment]
Right now it's roughly every 6 hours.

And it sounds like you might be interested in both https://planetarycomputer.microsoft.com/docs/quickstarts/stac-geoparquet/ and https://github.com/microsoft/PlanetaryComputer/issues/227 once it's implemented.

**Comment by edornd** [comment]
I see, that's certainly frequent enough for my use case, thanks! Also thank you for the extra pointers, much appreciated. The geoparquet part is neat, and the notifications could be awesome to react to new insertions, indeed. Thanks again!
